### PR TITLE
Add basic doc tests for Cards.

### PR DESCRIPTION
**Pull request by plural**

### CODE REVIEW

**Comment by plural** [comment]
Hey @NoahTheDuke - the fixtures are running on every test when I would like them to run once, ideally, for all tests, for the public data at least.

The materialized views are not populating here in the this test.

I am clearly not understanding the lifecycle here, so if you could help me clear things up here or in discord, it would be much appreciated!

File: spec/acceptance/card_spec.rb [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+  get "/api/v3/public/cards/:id" do
+    route_summary 'Retrieve a single Card by ID'
+    parameter :id, type: :string, required: true
+
+    let(:id) { 'hedge_fund' }
+    example_request "Get A Single Card" do
+      expect(status).to eq 200
+    end
+  end
```
Comment (NoahTheDuke) [inline spec/acceptance/card_spec.rb:27]: I don't really know how this testing framework works but this looks like a no-op test. I think that checking that an endpoint returns 200 is meaningless if it's returning the wrong thing.

**Comment by NoahTheDuke** [comment]
I don't really know how this doc testing framework works, but I think you can do:

```ruby
before(:suite) do
  #load test/fixtures here
end
```

**Comment by plural** [comment]
Oh, the test is just there to cause the docs to generate. I agree that
these tests are silly at the moment. :)

I'll give the other thing a shot this evening, thanks.

**Comment by plural** [comment]
Ok, did a couple more tweaks, but this is at least working how i would expect now and is faster - we load the fixtures once as part of db:seed in the test environment and then refresh the materialized views in `config.before(:suite)`

**Comment by NoahTheDuke** [comment]
That looks great! Nicely done.

**Review by NoahTheDuke — APPROVED** [review]

**Pull request merged**